spec/system/jobseekers/jobseekers_can_manage_a_profile_spec.rb

```diff
@@ -596,78 +567,38 @@
         field = find_field("Name of school or trust")
         field.fill_in(with: forbidden_organisation.name)
         click_on I18n.t("buttons.save_and_continue")
-      end
-
-      run_with_publisher(forbidden_publisher) do
-        visit publishers_jobseeker_profiles_path
-        expect(page).not_to have_content(profile.full_name)
-      end
 
-      run_with_publisher(permitted_publisher) do
-        visit publishers_jobseeker_profiles_path
-        expect(page).to have_content(profile.full_name)
-      end
-
-      run_with_jobseeker(jobseeker) do
-        visit jobseekers_profile_path
-        click_on I18n.t("jobseekers.profiles.hide_profile.summary.add_a_school")
+        choose "Yes", visible: false
+        click_on I18n.t("buttons.save_and_continue")
 
         field = find_field("Name of school or trust")
-        field.fill_in(with: permitted_organisation.name)
+        field.fill_in(with: forbidden_organisation.name)
         click_on I18n.t("buttons.save_and_continue")
-      end
 
-      run_with_publisher(permitted_publisher) do
-        visit publishers_jobseeker_profiles_path
-        expect(page).not_to have_content(profile.full_name)
+        expect(page).to have_content(I18n.t("jobseekers.profiles.hide_profile.schools.already_hidden", name: forbidden_organisation.name))
       end
 
       run_with_publisher(forbidden_publisher) do
         visit publishers_jobseeker_profiles_path
         expect(page).not_to have_content(profile.full_name)
+      end
 
-        visit publishers_jobseeker_profile_path(profile)
-        expect(page).to have_content("Page not found")
+      run_with_publisher(permitted_publisher) do
+        visit publishers_jobseeker_profiles_path
+        expect(page).to have_content(profile.full_name)
       end
 
       run_with_jobseeker(jobseeker) do
         visit schools_jobseekers_profile_hide_profile_path
-        within page.find(".govuk-summary-list__key", text: permitted_organisation.name).find(:xpath, "..") do
+        within page.find(".govuk-summary-list__key", text: forbidden_organisation.name).find(:xpath, "..") do
           click_on I18n.t("buttons.delete")
         end
         click_button I18n.t("jobseekers.profiles.hide_profile.delete.delete_school")
       end
 
-      run_with_publisher(permitted_publisher) do
```

Comment:
Prior to these changes this test:

1. If a jobseeker had not blocked organisations from seeing their jobseeker profile, then both organisations could see it.
2. a jobseeker could block one organisation, then checked that the blocked organisation could not see the jobseeker's profile, but that an organisation that wasn't blocked, could see it. 
3. It then it blocked the other one, and checked that both organisations could not see it.
4. It unblocked one organisation and checked that just one could see it but the other could not.

This change removes step 3 and in step 4 it only asserts that the unblocked organisation can be seen again in order to reduce run time for what (I think) is quite minimal benefit.